diff --git a/node_modules/electron-updater/node_modules/builder-util-runtime/out/httpExecutor.js b/node_modules/electron-updater/node_modules/builder-util-runtime/out/httpExecutor.js
--- a/node_modules/electron-updater/node_modules/builder-util-runtime/out/httpExecutor.js
+++ b/node_modules/electron-updater/node_modules/builder-util-runtime/out/httpExecutor.js
@@ -127,7 +127,7 @@
             });
             this.addErrorAndTimeoutHandlers(request, reject, options.timeout);
             this.addRedirectHandlers(request, options, reject, redirectCount, options => {
-                this.doApiRequest(options, cancellationToken, requestProcessor, redirectCount).then(resolve).catch(reject);
+                this.doApiRequest(options, cancellationToken, requestProcessor, redirectCount + 1).then(resolve).catch(reject);
             });
             requestProcessor(request, reject);
             onCancel(() => request.abort());
@@ -169,11 +169,11 @@
         const shouldRedirect = code >= 300 && code < 400;
         const redirectUrl = safeGetHeader(response, "location");
         if (shouldRedirect && redirectUrl != null) {
-            if (redirectCount > this.maxRedirects) {
+            if (redirectCount >= this.maxRedirects) {
                 reject(this.createMaxRedirectError());
                 return;
             }
-            this.doApiRequest(HttpExecutor.prepareRedirectUrlOptions(redirectUrl, options), cancellationToken, requestProcessor, redirectCount).then(resolve).catch(reject);
+            this.doApiRequest(HttpExecutor.prepareRedirectUrlOptions(redirectUrl, options), cancellationToken, requestProcessor, redirectCount + 1).then(resolve).catch(reject);
             return;
         }
         response.setEncoding("utf8");
@@ -252,7 +252,7 @@
             const redirectUrl = safeGetHeader(response, "location");
             if (redirectUrl != null) {
                 if (redirectCount < this.maxRedirects) {
-                    this.doDownload(HttpExecutor.prepareRedirectUrlOptions(redirectUrl, requestOptions), options, redirectCount++);
+                    this.doDownload(HttpExecutor.prepareRedirectUrlOptions(redirectUrl, requestOptions), options, redirectCount + 1);
                 }
                 else {
                     options.callback(this.createMaxRedirectError());
@@ -268,7 +268,7 @@
         });
         this.addErrorAndTimeoutHandlers(request, options.callback, requestOptions.timeout);
         this.addRedirectHandlers(request, requestOptions, options.callback, redirectCount, requestOptions => {
-            this.doDownload(requestOptions, options, redirectCount++);
+            this.doDownload(requestOptions, options, redirectCount + 1);
         });
         request.end();
     }
